### PR TITLE
[MU4] Allow ledgerlines exactly as wide as the noteheads

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
and fix the vtest bat files to use MuseScore4

Counterpart to #6359

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
vtest failures expected

File: vtest/gen-ref.bat [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 rem create reference
 
-set MSCORE=..\msvc.install_x64\bin\musescore.exe
```
Comment (vpereverzev) [inline vtest/gen-ref.bat:4]: @igorkorsukov take a look on that

Comment (Jojo-Schmitz) [reply vtest/gen-ref.bat:4]: oops

Comment (Jojo-Schmitz) [reply vtest/gen-ref.bat:4]: Was wrong before, and still ids ;-)

Comment (Jojo-Schmitz) [reply vtest/gen-ref.bat:4]: It is a much simpler script and not meant to be run automatically, and so %INSTALL_PATH% is not set, so I guess we can let this pass for now?

Comment (igorkorsukov) [reply vtest/gen-ref.bat:4]: I think yes, we can pass this 
This is files not used on CI, as I understand, they for local testing on Windows?

**Review by vpereverzev — APPROVED** [review]

File: vtest/gen-ref.bat (hunk 1 above)
Comment (Jojo-Schmitz) [reply vtest/gen-ref.bat:4]: Yes

File: vtest/gen-ref.bat [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+set MSCORE=..\msvc.install_x64\bin\musescore4.exe
 set DPI=130
 
 %MSCORE% %1.mscz -r %DPI% -o %1.png
```
Comment (Jojo-Schmitz) [inline vtest/gen-ref.bat:7]: Also here it needs to be `%MSCORE% %1.mscx -r %DPI% -o %1.png`, mscx vs. mscz, as I just found out

**Pull request merged**